src/api/cvc4cpp.h

```diff
@@ -2868,12 +2868,16 @@ class CVC4_PUBLIC Solver
    */
   Term simplify(const Term& t);
 
+  Term expandDefinitions(Term t) const;
```

Comment:
This needs to be documented.

Reply:
> Do we really need to export `expandDefinitions`? I feel like this should be internal only.

I discussed this with Andy and he agrees. I removed this change. In addition, I will remove `ExpandDefinitionsCommand` class and `--expand-definitions` option in a separate PR.